include/natalie/io_object.hpp

```diff
@@ -61,9 +62,10 @@ class IoObject : public Object {
     int fsync(Env *);
     Value getbyte(Env *);
     Value gets(Env *) const;
-    Value initialize(Env *, Value, Value = nullptr);
+    Value initialize(Env *, Value, Value = nullptr, Value = nullptr, Value = nullptr, Value = nullptr, Value = nullptr, Value = nullptr, Value = nullptr, Value = nullptr, Value = nullptr, Value = nullptr);
```

Comment:
Good grief, lol

Reply:
I'm open to an alternative way of doing the bindings if you have any ideas. 😅

Reply:
My plan is to change the bindings to `argc: 1..`, so the complete `Args` objects is forwarded to `IoObject::initialize`, and from there forward it to some helper function to validate the number of elements and extract the keyword arguments.

Reply:
I dig it